### PR TITLE
Open register page after a clean installation

### LINKED ISSUE #902: Open register page after a clean installation
After a clean installation, if no account has been created, we should open the /register page, even if APP_DISABLE_SIGNUP is set to true.
That makes the ability to setup the environnement database without creating a new account, and let the user create his first account by using the real form.

This should be usefull for docker / heroku / any closed environnement.

### PR DESCRIPTION
**Pull request by asbiin**
This close #902.

#### Before submitting the PR
- [x] Read the [CONTRIBUTING document](https://github.com/monicahq/monica/blob/master/CONTRIBUTING.md) before submitting your PR.
- [x] If the PR is related to an issue or fix one, don't forget to indicate it.
- [x] Make sure that the change you propose is the smallest possible.
- [ ] Screenshots are included if the PR changes the UI.
- [ ] If you change the UI, make sure the user experience is consistent with the current interface.

#### Code-related tasks
- [x] Tests added for this feature/bug.
- [x] Impact on the seeders.
- [ ] Impact on the API.

#### If the code changes the SQL schema
- [ ] Impact on account export.
- [ ] Impact on importing data with `vCard` and `.csv` files.
- [ ] Impact on account reset and deletion.

#### Other tasks
- [x] [CHANGELOG](https://github.com/monicahq/monica/blob/master/CHANGELOG) entry added, if necessary, under `UNRELEASED`.
- [x] [CONTRIBUTORS](https://github.com/monicahq/monica/blob/master/CONTRIBUTORS) entry added, if necessary.
- [ ] If it's relevant, add the documentation about your feature in the README file.
- [ ] Indicate `[wip]` in the title of the PR it is is not final yet. Remove `[wip]` when ready. Otherwise the PR will be considered complete and rejected if it's not working.

### CODE REVIEW

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testBasicExample-0.png
[id:YzNUwmdEHznLQzB]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testBasicExample-0.png
[id:aYX13uViPci7hGQ]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testBasicExample-0.png
[id:oANqHieT8q3hjWB]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testBarCodeContent-0.png
[id:LE8m7s5cYoDDsmP]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2fa-0.png
[id:yaFPhF1gKeSlE1n]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2faDisable2fa-0.png
[id:sY7Hq9ufPnxE4T9]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2faDisable2faWrongCode-0.png
[id:HtaADX0g3Iaip3d]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2faLogin-0.png
[id:spWZHMYzUJ6ypNI]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2faLoginWrongCode-0.png
[id:cSipmy19BflzY3E]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2faWrongCode-0.png
[id:XlqXm5xF1KZV3Wj]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testHas2faEnableBarCode-0.png
[id:l6rQx3OuBtiSYYh]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-test_import_button_leads_to_import_screen-0.png
[id:p5uV96a5wVXjk8F]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-test_upload_vcard_is_accessible_from_add_contact_view-0.png
[id:sPvD6r54drjY21B]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-test_user_can_import_contacts_from_a_vcf_card-0.png
[id:ruRuvgLjQSqL8Aj]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-test_user_see_error_when_importing_broken_vcard-0.png
[id:8zXO3ycazZNcQ81]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testBarCodeContent-0.png
[id:xOLkLL6HAceI5od]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2fa-0.png
[id:mHfH8pIKhNaE5FD]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2faDisable2fa-0.png
[id:gYbNDEsQkZmUUwn]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2faDisable2faWrongCode-0.png
[id:75bTX9YKVSL2hKL]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2faLogin-0.png
[id:FHbxNRPd7TlSrnz]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2faLoginWrongCode-0.png
[id:nvtfWAYLWcVVNg6]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2faWrongCode-0.png
[id:IQ5ptdiMd1dvHY3]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testHas2faEnableBarCode-0.png
[id:9r1LtwbkR0RMSab]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-test_import_button_leads_to_import_screen-0.png
[id:UhuMx6OCUmHs3Cj]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-test_upload_vcard_is_accessible_from_add_contact_view-0.png
[id:kTzhJeevsyykIft]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-test_user_can_import_contacts_from_a_vcf_card-0.png
[id:eZQwT1rhALYXbZM]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-test_user_see_error_when_importing_broken_vcard-0.png
[id:2OqAuh7Z1evXAQY]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testBarCodeContent-0.png
[id:CqdO1VglXUK1wRr]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2fa-0.png
[id:QPEDe1UIubQoW66]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2faDisable2fa-0.png
[id:KNLcnUvtTVsZ1YU]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2faDisable2faWrongCode-0.png
[id:MwfGQNIobwTVm8G]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2faLogin-0.png
[id:FEmNg230acUymRM]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2faLoginWrongCode-0.png
[id:rZfsrI9xnhq5EzZ]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2faWrongCode-0.png
[id:oXInJ1dErh9Zofv]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testHas2faEnableBarCode-0.png
[id:wEn89zc6Ku8ZzHO]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-test_import_button_leads_to_import_screen-0.png
[id:8eXGqAGLaYn6P9s]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-test_upload_vcard_is_accessible_from_add_contact_view-0.png
[id:2gMu2883lWnxcpb]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-test_user_can_import_contacts_from_a_vcf_card-0.png
[id:2mR9I9GVGEuktek]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-test_user_see_error_when_importing_broken_vcard-0.png
[id:KtQXVF2GrIno9HM]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testBarCodeContent-0.png
[id:ZJ6GrEQaqrAs8Yk]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2fa-0.png
[id:NV5WxEVcwWGzV4C]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2faDisable2fa-0.png
[id:FAeq5xaDPPx3BAJ]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2faDisable2faWrongCode-0.png
[id:7mVfcmQ8J0vWLKR]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2faLogin-0.png
[id:9Twu2FzuiXAIGRN]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testBarCodeContent-0.png
[id:KQ0DvYLzKhHQUCn]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2faLoginWrongCode-0.png
[id:fvBjr9YIl9r3GyI]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2fa-0.png
[id:GLBwEYd5IpWECqB]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2faWrongCode-0.png
[id:pX0V9aj8PYPVcaO]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testHas2faEnableBarCode-0.png
[id:l3WWkRzxaUGtEz4]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2faDisable2fa-0.png
[id:4FDTs1JONlahUqh]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-test_import_button_leads_to_import_screen-0.png
[id:JqAjVpEWHk3rmtr]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2faDisable2faWrongCode-0.png
[id:YPlNUn6zzrB9sXf]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-test_upload_vcard_is_accessible_from_add_contact_view-0.png
[id:hA6XQ7zwrGZAxxS]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2faLogin-0.png
[id:YwJQVdVywwRyrFc]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-test_user_can_import_contacts_from_a_vcf_card-0.png
[id:093VQHPr54BjbRC]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2faLoginWrongCode-0.png
[id:aW9A5h7sZePZBeV]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-test_user_see_error_when_importing_broken_vcard-0.png
[id:kRYoXgdv01DvBbu]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2faWrongCode-0.png
[id:Nuvsoohxo47isXW]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testHas2faEnableBarCode-0.png
[id:1wVdFMjqqmGe4QN]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-test_upload_vcard_is_accessible_from_add_contact_view-0.png
[id:RhHF0OgaDQs8JKi]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-test_user_can_import_contacts_from_a_vcf_card-0.png
[id:iyoWHK8mDHNPwcY]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-test_user_see_error_when_importing_broken_vcard-0.png
[id:ZvXSlWzbD7z2sKe]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testBarCodeContent-0.png
[id:Sp3IlpSzvMGoo1Z]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2fa-0.png
[id:hnIoSJZWQhb6M1y]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2faDisable2fa-0.png
[id:MbZSlYPzRwM0Gxp]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2faDisable2faWrongCode-0.png
[id:AwQq6yxewl5NNmo]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2faLogin-0.png
[id:Tr6LdsgJHOJyb58]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2faLoginWrongCode-0.png
[id:ArmMmpxHYa5eKNl]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testEnable2faWrongCode-0.png
[id:eK8hRfVXZEIpVXR]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-testHas2faEnableBarCode-0.png
[id:fbWddhtRYpJNeOl]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-test_import_button_leads_to_import_screen-0.png
[id:n2VV0UzggCxMwEW]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-test_upload_vcard_is_accessible_from_add_contact_view-0.png
[id:L4xobfIDncJLD2v]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-test_user_can_import_contacts_from_a_vcf_card-0.png
[id:JD1i0tUYozEdUQj]

**Comment by asbiin** [comment]
[test]
capture
tests/Browser/screenshots/failure-test_user_see_error_when_importing_broken_vcard-0.png
[id:io0nPYmWaAACOqY]

File: resources/views/auth/login.blade.php [hunk 1]
```diff
@@ -40,6 +40,8 @@
                   <li>Forget your password? <a href="/password/reset">Reset your password</a></li>
                   @if(! config('monica.disable_signup'))
                     <li>Don't have an account? <a href="/register">Sign up</a></li>
+                  @elseif(! \App\Account::hasAny())
+                    <li>Create the first account by <a href="/register">Sign up</a></li>
```
Comment (djaiss) [inline resources/views/auth/login.blade.php]: `By signing up`

File: resources/views/auth/register.blade.php [hunk 2]
```diff
@@ -8,8 +8,13 @@
 
           <div class="signup-box">
             <img class="logo" src="/img/small-logo.png" alt="">
-            <h2>Create your Monica account</h2>
-            <h3><a href="/login">Log in</a> if you already have an account.</h3>
+            @if($first)
+              <h1>Welcome to your newly installed Monica instance</h1>
+              <h2>You need to create a Monica account</h2>
```
Comment (djaiss) [inline resources/views/auth/register.blade.php]: You need to create an account to use Monica

**Pull request merged**